.github/workflows/deploy.yml

```diff
@@ -10,6 +10,8 @@ on:
 jobs:
   deploy:
     runs-on: ubuntu-latest
+    env:
+      SLS_DEPRECATION_DISABLE: "*" # Turn off deprecation warnings in the pipeline
```

Comment:
This disables severless deprecation notices, which tend to clutter the build output.  This change is purely cosmetic.  Local deployments will continue to see deprecation warnings.